lib/asciidoctor/pdf/text_transformer.rb

```diff
@@ -6,7 +6,7 @@ module TextTransformer
       XMLMarkupRx = /&#?[a-z\d]+;|</
       PCDATAFilterRx = /(&#?[a-z\d]+;|<[^>]+>)|([^&<]+)/
       TagFilterRx = /(<[^>]+>)|([^<]+)/
-      WordRx = /\S+/
+      WordRx = /[[:word:]]+/
```

Comment:
In Asciidoctor, we use the character properties instead of the POSIX character classes since the former are Unicode-friendly. Therefore, I will change this to `\p{Word}`.